index.md

```diff
@@ -7,14 +7,15 @@ layout: home
 
 Over the last decade, [OCI containers](https://specs.opencontainers.org/image-spec/) have become a de facto way to deploy a complete functioning Linux user space as an application.
 A large set of practices and tooling have evolved around them.
-Bootable containers are a modern opinionated way of deploying, configuring and managing immutable image based Linux systems.
+Bootable containers are a modern opinionated way of deploying, configuring and managing image based Linux systems.
```

Comment:
Thanks for doing this!

It's a challenge since I think the usage of "immutable" is widespread in the industry still years later. My high level thoughts is that it's a word that is only useful at a *very* introductory level but once you get past the surface it can be more misleading than helpful.

So :+1: to dropping it here.

Reply:
Atomic / Image-based Linux systems